### DISCUSSION TITLE
I like this...

### DISCUSSION THREAD

**Discussion by Yoshi-RBLX** [mayo78/FNF-PsychEngine-BambiAddon#3] [General]
This engine is really cool, even for v1. While it lacks some things like freeplay categories and some other stuff it's still really neat. I want to use this for a mod of mine.

**Comment by mayo78** [comment]
you can add freeplay categories yourself with a custom state, returning Function_Stop in onSelect in MainMenuState you can send the player to a custom freeplay select state

**Reply by Yoshi-RBLX** [reply]
> you can add freeplay categories yourself with a custom state, returning Function_Stop in onSelect in MainMenuState you can send the player to a custom freeplay select state

BTW I had a question. I'm not that good of a coder, not even one at all to be honest, so how do I add in the glitch shader into the lua stages, because I really don't know how...
